test/autoCompletion.test.ts

```diff
@@ -96,8 +98,8 @@ describe('Auto Completion Tests', () => {
             },
           },
         });
-        const content = 'na|:|';
-        const completion = parseSetup(content);
+        const content = 'na'; // len: 2
+        const completion = parseSetup(content, 2);
```

Comment:
I have added a comment to every `content` string stating its `length`
This is so that the maintainer has a quick understanding of the `position` value in the next call.
In this case, `position === 2` which is beyond the `content.length`
So we cannot use the new algorithm, so we leave the old one in place.